### PR TITLE
Working/esconfig 1

### PR DESCRIPTION
**Pull request by IT-WIBRC**

### CODE REVIEW

**Comment by IT-WIBRC** [comment]
check the rule about `[vue/no-multiple-template-root]
The template root requires exactly one element.eslint-plugin-vue`

**Comment by WIBRC45920** [comment]
Yes I have the same problem. If you find a solution please I want to know how to resolve it.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+  npm install -D repository#branch
+```
+
+## 2. Extend the configuration
+create the `.eslintrc` file in the root of your project. after copy this in the file:
+
+```js
+{
+  extends: "@wibrc/eslint-config-vue"
+}
```
Comment (WIBRC45920) [inline README.md:31]: Please add how to work with on some IDE as VsCode, WebStorm, and others.

File: package.json [hunk 2]
```diff
@@ -1,62 +1,47 @@
 {
   "name": "@wibrc/eslint-config-vue",
   "version": "0.0.1",
-  "description": "reusable and sharable `eslint` configuration",
+  "description": "reusable and sharable `eslint` configuration for vue project",
```
Comment (WIBRC45920) [inline package.json:4]:
```suggestion
  "description": "A reusable and shareable `eslint` configuration for Vue project",
```

File: package.json [hunk 3]
```diff
@@ -1,62 +1,47 @@
...
-    "stylelint-scss": "~4.1.0",
-    "lint-staged": "12.1.2"
+    "lint-staged": "12.1.2",
+    "prettier": "2.8.3",
+    "stylelint": "14.2.0",
+    "eslint": "8.32.0"
   },
   "dependencies": {
-    "eslint": "~8.32.0",
+    "@rushstack/eslint-patch": "~1.1.0",
```
Comment (WIBRC45920) [inline package.json:28]: I think you can change `~` into `^` here.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,32 @@
 # wibrc-eslint-config-vue
 An eslint shareable configuration for Vuejs app
+## 1. Install since github as dev dependency
```
Comment (WIBRC45920) [inline README.md:3]: Please make a description of this configuration. Read how to write a good and professional `README` file. I think this are going to be good for you for the next projects.

**Comment by IT-WIBRC** [comment]
> check the rule about `[vue/no-multiple-template-root] The template root requires exactly one element.eslint-plugin-vue`

This is due to the fact that your project is in another parent file like `parent/project`. And the `vetur` extension is going to analyse the parent file and don't see the `package.json` file. So to resolve it in this case, you can just  open the `project` itself on vscode or change the `vetur` configuration as in this link `https://stackoverflow.com/questions/64867504/vue-3-the-template-root-requires-exactly-one-element-eslint-plugin-vue`

File: package.json (hunk 3 above)
Comment (IT-WIBRC) [reply package.json:28]: I tried to show the difference and i found this: https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json#comment53973834_22345808.

That is not a bad idea but i'm going to think about that.

**Review by WIBRC45920 — APPROVED** [review]

File: .eslintrc.js [hunk 5]
```diff
@@ -13,6 +13,8 @@ module.exports = {
   env: {
     jquery: true,
     node: true,
+    browser: true,
+    jest: true,
```
Comment (WIBRC45920) [inline .eslintrc.js:17]: Nice here

**Review by WIBRC45920 — APPROVED** [review]

**Review by WIBRC45920 — APPROVED** [review]

**Review by WIBRC45920 — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,2 +1,171 @@
...
+
+You might like them - or you might not. Don't worry you can always change them.
+
+## What it does
+- Lints JavaScript and TypeScript based on the latest standards
+- Fixes issues and formatting errors with Prettier
+- Lints + Fixes inside of html script tags
+- You can see all the [rules here](https://github.com/IT-WIBRC/wibrc-eslint-config-vue/blob/master/.eslintrc.js). You are very welcome to overwrite any of these settings, or just fork the entire thing to create your own.
+
+## Project install
```
Comment (WIBRC45920) [inline README.md:13]: Nice you have arranged the `readme`

**Review by WIBRC45920 — APPROVED** [review]
Ok it's nice now

File: README.md [hunk 7]
```diff
@@ -1,2 +1,171 @@
...
+
+   ```js
+    {
+      extends: "@wibrc/eslint-config-vue"
+    }
+   ```
+ 
+   3. You can add two scripts to your package.json to lint and/or fix:
+
+   ```json
```
Comment (WIBRC45920) [inline README.md:59]: Did you change the alignment to resolve this problem, pleas?

File: README.md [hunk 8]
```diff
@@ -1,2 +1,171 @@
...
+  It should work as above.
+
+  ## With JetBrains Products (IntelliJ IDEA, WebStorm, RubyMine, PyCharm, PhpStorm, etc)
+
+  If you have previously configured ESLint to run via a File Watcher, [turn that off.](https://www.jetbrains.com/help/idea/using-file-watchers.html#enableFileWatcher)
+
+  ### If you choose Local / Per Project Install Above
+
+  1. Open ESLint configuration by going to File > Settings (Edit > Preferences on Mac) > Languages & Frameworks > Code Quality Tools > ESLint (optionally just search settings for "eslint")
+  1. Select **Automatic ESLint Configuration**
```
Comment (WIBRC45920) [inline README.md:151]: Change this in ordered list, please

File: README.md [hunk 9]
```diff
@@ -1,2 +1,171 @@
...
+  1. Check **Run eslint --fix on save**
+
+  ### If you choose Global Install
+
+  The following steps are for a typical Node / ESLint global installtion. If you have a customized setup, refer to JetBrains docs for more [ESLint Configuration Options](https://www.jetbrains.com/help/webstorm/eslint.html#ws_js_eslint_manual_configuration).
+
+  1. Open ESLint configuration by going to File > Settings (Edit > Preferences on Mac) > Languages & Frameworks > Code Quality Tools > ESLint (optionally just search settings for "eslint")
```
Comment (WIBRC45920) [inline README.md:158]: Same here and below

**Review by WIBRC45920 — CHANGES_REQUESTED** [review]

**Review by WIBRC45920 — APPROVED** [review]

File: README.md [hunk 10]
```diff
@@ -1,2 +1,171 @@
...
+  2. Select **Manual ESLint configuration**
+  3. Choose your **Node interpreter** from the detected installations
+  4. Select the global **ESLint package** from the dropdown
+  5. Leave Configuration File as **Automatic Search**
+  6. Check **Run eslint --fix on save**
+
+  ### Ensure the Prettier plugin is disabled if installed.
+
+  1. Open Prettier configuration by going to File > Settings (Edit > Preferences on Mac) > Languages & Frameworks > Code Quality Tools > Prettier (optionally just search settings for "prettier")
+  2. Uncheck both **On code reformat** and **On save**
```
Comment (WIBRC45920) [inline README.md:168]: Thanks

**Review by WIBRC45920 — APPROVED** [review]

**Review by WIBRC45920 — APPROVED** [review]
Nice it's all.

**Review by WIBRC45920 — APPROVED** [review]

**Pull request merged**